SECURITY.md

```diff
@@ -0,0 +1,46 @@
+# Security Notice
+
+## What?
+
+This is the security notice for all GDS (Government Digital Service) `alphagov` repositories. The notice explains how vulnerabilities should be reported to GDS. At GDS there are cyber security and information assurance teams, as well as security-conscious people within the programmes, that assess and triage all reported vulnerabilities.
+
+## Reporting a Vulnerability
+
+GDS is an advocate of responsible vulnerability disclosure. If you’ve found a vulnerability, great! We would like to know so we can fix it.
```

Comment:
I don't like the "great!" bit. It's really not great at all. I think just join the two sentences:

> If you’ve found a vulnerability, we would like to know so we can fix it.

Reply:
I guess it's not great that the code has a vulnerability, but it is great that someone found it and wants to report it to us. For clarity though, I agree, we should remove the `great!`.